pages/api_integration-trading/isolated_markets.md

```diff
@@ -0,0 +1,66 @@
+# Isolated Markets
+
+In v5.0.0 the Isolated Markets feature was added to the V4 chain software. The below is an overview of how trading will work on Isolated Markets on the V4 chain software.
+
+>Note: This document covers how the feature works from the protocol point of view and not the front-end or the indexer.
+
+# Querying for Isolated Markets
+
+There is a new `market_type` parameter in the `PerpetualParams` proto struct that indicates the type of market. 
+
+There are 2 possible values for this parameter:
+
+- `PERPETUAL_MARKET_TYPE_CROSS` - markets where subaccounts can have positions cross-margined with other `PERPETUAL_MARKET_TYPE_CROSS` markets, all markets created before the v5.0.0 upgrade are `PERPETUAL_MARKET_TYPE_CROSS` markets
+- `PERPETUAL_MARKET_TYPE_SOLATED` - markets where subaccounts can only have a single position in, no cross-margining is possible
```

Comment:
Maybe describe this as
"Markets that can only be margined in isolated. No cross-margining with other markets is possible"